js/build-directory.js

```diff
@@ -16,13 +16,23 @@ function html_entity_decode(html) {
 }
 
 function splitByCommas(str) {
+  if (Array.isArray(str)) {
+    return str;
+  }
+
   if (typeof str !== 'string') {
```

Comment:
I don't get this. If it's not a string (and not an array because of the above condition), what is this expected to be? an object? a number? and if so, why?

Reply:
@squallstar Yes, it could be a number. If the number `3.14` got passed, then it's possible that it got passed as a single value. I was testing this and found that my brain reacted when `splitByCommas(3.14)` returned an empty array `[]`. The thought is, "if I gave it some kind of input, why has the function eaten away my input and returned nothing?"

So, I gathered it's more reasonable to return the input in an array so that the input remains.